asset_dashboard/forms.py

```diff
@@ -87,3 +89,50 @@ class Meta:
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
+
+
+class LocalAssetForm:
```

Comment:
If you agree with going to DRF route, the other comments on this file are moot.

Reply:
I originally did that, but the django form class have a bunch of html that isn't necessary for saving the assets since we're accepting via ajax and not a standard html form. I had to override those required fields in the init function and it didn't feel right. I was thinking this class was more like a serializer for validating the data, but technically it's like a form too. So it's in this weird limbo -- I think this is where DRF would come in handy?